src/crypto/verification/request/ToDeviceChannel.js

```diff
@@ -356,4 +356,12 @@ export class ToDeviceRequests {
             }
         }
     }
+
+    getRequestsInProgress(userId) {
+        const requests = this._requestsByUserId.get(userId);
```

Comment:
Maybe name this `requestsByTxnId` like other functions do? It's not obvious from reading this one by itself that there are two level of maps flying around here.

Reply:
mm, good point